src/apps/service/apps/de/listings.clj

```diff
@@ -400,8 +420,9 @@
   [user workspace group-id perms {:keys [public-app-ids] :as params}]
   (when-let [format-fns (virtual-group-fns group-id)]
     (let [app-listing ((:format-listing format-fns) user workspace params)
-          format-app  (get-app-listing-formatter user false perms (map :id app-listing) public-app-ids)]
-      (-> ((:format-group format-fns) user workspace params)
+          app-ids     (map :id app-listing)
+          format-app  (get-app-listing-formatter user false perms app-ids public-app-ids)]
+      (-> ((:format-group format-fns) user workspace (assoc params :listing-ids app-ids))
```

Comment:
Adding the listing ids to `params` was the only way I could think of to get the total count for formatting. The other virtual categories seem to have their own DB query that they run to get the count, but I didn't think I could do that since I need the blessed-ids from permissions and I already have that list here anyway.  Very open to suggestions if this seems off.